docs/lang/articles/c-api/taichi_core.md

```diff
@@ -325,7 +325,6 @@ A collection of Taichi kernels (a compute graph) to launch on the offload target
 ```c
 // enumeration.error
 typedef enum TiError {
-  TI_ERROR_TRUNCATED = 1,
```

Comment:
Success codes feel unnecessary at this point so it's removed and are confusing among these error codes.